components/TVChart/TVChart.tsx

```diff
@@ -82,7 +82,17 @@ export function TVChart({
 		return () => {
 			clearExistingWidget();
 		};
-	}, [baseCurrencyKey, quoteCurrencyKey]);
+	}, [
+		baseCurrencyKey,
+		quoteCurrencyKey,
+		autosize,
+		containerId,
+		fullscreen,
+		interval,
+		libraryPath,
+		overrides,
+		studiesOverrides,
```

Comment:
Adding these causes some flickering on the `TVChart` component when it is refocused.

Reply:
I've removed these and added // eslint-disable-next-line react-hooks/exhaustive-deps